dbcrossbar/src/drivers/gs/write_remote_data.rs

```diff
@@ -61,6 +61,17 @@ pub(crate) async fn write_remote_data_helper(
         .job_labels
         .to_owned();
 
+    let job_project_id = source_args
+        .driver_args()
+        .deserialize::<GCloudDriverArguments>()
+        .context("error parsing --from-args")?
+        .job_project_id
+        .to_owned();
```

Comment:
We should only pass `GCloudDriverArguments` once here, not twice. You can assign it to a variable as follows:

```rust
    let driver_args = source_args
        .driver_args()
        .deserialize::<GCloudDriverArguments>()
        .context("error parsing --from-args")?;
```

...and then grab `job_labels` and `job_project_id` separately. The driver arg extraction process is fairly expensive (not that it matters here). More importantly, it's more consistent with the other drivers, and it's good practice.